scripts/generate-global-changelog.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {execSync} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const IGNORE_LIST = [
+	'packages/demos/CHANGELOG.md',
+	'clayui.com/CHANGELOG.md',
+	'CHANGELOG.md',
+];
+
+const dateObj = new Date();
+const month = dateObj.getMonth() + 1;
+const day = dateObj.getDate();
+const year = dateObj.getFullYear();
+
+const dateString = `${year}-${month}-${day}`;
+
+const globalChangeLog = path.join(__dirname, '../CHANGELOG.md');
+
+function main() {
+	const diffOutput = execSync(
+		'git diff HEAD^ --name-only -- "*CHANGELOG.md"'
+	).toString();
```

Comment:
Are there any sanity-checks that we can/should add here? Like, print a warning if the worktree is dirty, or if there are no changes? etc (I don't have any specific ideas here, but just trying to think of possibly "failure modes" where the script is called at the wrong time or in the wrong way.)

Reply:
I added a check to make sure we actually have changelogs that have changed.

We expect the tree to be dirty since this script is called AFTER changelogs are updated but BEFORE they are commited